gpii/node_modules/userListeners/src/listeners.js

```diff
@@ -38,6 +37,11 @@ fluid.defaults("gpii.userListeners", {
         }
     },
     components: {
+        // Note: The subcomponent "lifecycleManager" is distributed from flowManager so it doesn't disturb
+        // user listeners unit tests.
+        // lifecycleManager: {
```

Comment:
I think it is reasonable and also clearer to have this definition uncommented in that it emphasises that there really should always be some kind of lifecycleManager here. If we aren't interested in mocking the LifecycleManager in usbTests.js we can then just override this with a distribution of fluid.emptySubcomponent to this point - this is consistent with the rest of the strategy in "gpii.tests.userListener.usbListener" in that it is expecting to be overriding existing definitions and cutting linkages. 
While you are here, could you tidy up gpii.tests.userListener.getUSBListener so that the extra configuration takes the form of a grade.
To be honest, we are testing so little of the linkage here that it might be worth doing away with usbTests.js use of "gpii.userListeners" and just giving it a single gpii.userListeners.usb instance